### ISSUE TITLE
Pullquote: Improve customization options

### ISSUE THREAD

**Issue by beafialho** [WordPress/gutenberg#48257]
<img width="1512" alt="Captura de ecrã 2023-02-20, às 15 42 17" src="https://user-images.githubusercontent.com/65220155/220152426-c7595002-87f7-45f9-ab59-2f634826f090.png">

Using Global Styles currently, it isn't possible to:
- Customize the quote and citation separately
- Change text alignment

 To achieve a design like this or similar:

<img width="937" alt="Captura de ecrã 2023-02-20, às 15 58 25" src="https://user-images.githubusercontent.com/65220155/220153196-d8f3fc10-f8e4-426f-b544-1a65c0464ee2.png">

**Comment by jasmussen** [comment]
Good issue, thank you.

If we're going to have a separate _Pullquote_ block, it needs at least the same level of customization as the Quote block does.

I'm not necessarily suggesting the pullquote block be deprecated, but in terms of having two quote blocks, there's some duplication going on, which was more necessary in the past where we had fewer styling options. But with more advanced styling options available to the quote block itself today, and in terms of consolidating and simplifying, would you be able to accomplish the same using just the quote block and the design tools available there?

To phrase the question here: what is it about the Pullquote block that you feel is missing from the regular Quote block?

**Comment by beafialho** [comment]
> If we're going to have a separate Pullquote block

Just to clarify, the Pullquote block is currently edited separately from the Quote block in the Style Book. Is that what you meant?

AFAIK quote and pullquote are meant to be different things so I understand it as expected that they're styled separately. However, if we had a way to **create and save several different quote styles**, maybe two different blocks wouldn't make sense or be necessary. It could be as easy as creating any design as a paragraph and have an option to "Save as Quote Style" and name it.

![save](https://user-images.githubusercontent.com/65220155/234833636-7c964071-a82e-4685-92b2-446b8fe65bae.png)

Then we could see it here:

<img width="278" alt="Captura de ecrã 2023-04-27, às 11 05 38" src="https://user-images.githubusercontent.com/65220155/234830827-888772b4-e191-4671-b2a4-689dbf7a5985.png">

Today, both the quote and pull quote have limited customisation options. Particularly in the citation bits and borders. Here is an example:

https://user-images.githubusercontent.com/65220155/234833295-a0e57aec-35a4-419e-8fad-c7b02dc0e6f0.mp4

Is this helpful?

**Comment by jasmussen** [comment]
> Just to clarify, the Pullquote block is currently edited separately from the Quote block in the Style Book. Is that what you meant?

Yes, I mean: what if we didn't have a pullquote block at all, only the Quote block.

Yes this is definitely helpful, and the ability to save and customize style variations seems like a useful way to potentially move this forward. What does @WordPress/gutenberg-design think?

* Should we have Quote and Pullquote blocks and employ the same full stylability of both, adding currently missing features to the pullquote block?
* Should the Pullquote block eventually be retired or deprecated in favor of block style variations a developer can define and design in theme.json?

I'm leaning towards _keeping_ the pullquote block, as it's useful to have the creative expression surfaced right in the inserter. It also feels like the most pragmatic and basic way forward. Finally it doesn't preclude us from still adding customizable style variations to both blocks.

**Comment by beafialho** [comment]
I'm also ok with keeping both blocks, as long as we're able to remove and/or customise the elements that can't currently be changed (the left border and the citation size, appearance, etc).

**Comment by jameskoster** [comment]
Please correct me if I am mistaken, but isn't the key differentiator between regular quotes and pullquotes positioning? IE regular quotes flow with the rest of the copy, while pullquotes are somewhat extracted from that flow to give visual emphasis:

<img width="1086" alt="Screenshot 2023-04-27 at 14 25 23" src="https://user-images.githubusercontent.com/846565/234876126-f1bbb914-5f33-489a-acf4-87a7ce01935a.png">


Given the block doesn't affect the position at all, I'm wondering if we really need both? It also seems reasonable to apply the same style to both 'types' of quotes which would remain tricky if they're separate blocks.

**Comment by beafialho** [comment]
My understanding is that quotes separate a section of text usually quoted from an outside source that's relevant to the article, while pull quotes are text pulled from the article and so duplicated, but presented as an attention-grabbing visual element.

**Comment by jasmussen** [comment]
If we're going to have a pullquote block, it seems valid enough to have the same styling options as the quote block has, at least. So I swapped out some labels.

**Comment by t-hamano** [comment]
Please note that we have deprecated the Pullquote block. See #11610 for more details.

**Comment by jasmussen** [comment]
Let's close this one then.

**Comment by t-hamano** [comment]
The Pullquote block has been restored in #75122, so let's reopen this issue.
